### PR TITLE
Show past rank when setting new one

### PR DESCRIPTION
**Pull request by BrandonWLee**

### CODE REVIEW

**Comment by BrandonWLee** [comment]
Wait, didn't grab discordUser

File: commands/rank.js [hunk 1]
```diff
@@ -51,8 +51,13 @@ exports.run = async (client, message, args, level) => {
           userId: key,
           rank: rank
         });
-
-        channel.send(`${message.author.username}'s rank has been set to: ${rank}`);
+       // retrieve data for user
+        const oldRank = client.ranks.get(discordUser.id);
+        if (oldRank) {
+          channel.send(`${message.author.username}'s rank has been set to: ${rank}. Was ${oldRank}`);
```
Comment (mynameisivan) [inline commands/rank.js:57]: minor: can we changed was updated from x to y?

**Review by mynameisivan — APPROVED** [review]

File: commands/rank.js (hunk 1 above)
Comment (BrandonWLee) [reply commands/rank.js:57]: Sure thing buddy

Comment (BrandonWLee) [reply commands/rank.js]: done

**Pull request merged**